### ISSUE TITLE
字幕的选择希望能智能一点

### ISSUE THREAD

**Issue by kneep** [iina/iina#3115]
1. 至少已经播放过的文件如果重新播放，应该自动切到上次播放时选择的字幕，目前好像只会选择默认的字幕
2. 建议根据系统语言来自动选择字幕（如果字幕轨道中带有语言信息的话）
谢谢

**Comment by veigarwang** [comment]
非常赞同！

**Comment by kid-oh** [comment]
希望默认加载同目录下同名的字幕，正是因为没字幕才去下载字幕的，不然每次都得手动调字幕很烦

**Comment by lhc70000** [comment]
@kid-oh 请检查偏好设置 - 字幕中的「自动加载」，默认情况下至少包含视频文件名的字幕是会自动加载的

**Comment by kid-oh** [comment]
> @kid-oh 请检查偏好设置 - 字幕中的「自动加载」，默认情况下至少包含视频文件名的字幕是会自动加载的

但是不会默认选择，还需要手动切换一下

**Comment by wj4901** [comment]
是的 不支持按照语言偏好优先打开对应语言的字幕,而且也不支持优先打开文件夹同名的字幕,重新播放后不能加载上次手动切换字幕.无论如何感觉字幕智能选择这块还是做得不够好
